SYCL/XPTI/basic_event_collection.cpp

```diff
@@ -0,0 +1,125 @@
+// REQUIRES: xptifw, opencl
+// RUN: %clangxx %s -DXPTI_COLLECTOR -L%xptifw_lib -I%xptifw_includes -shared -fPIC -std=c++17 -o %t_collector.dll -lxptifw
+// RUN: %clangxx -fsycl -fsycl-unnamed-lambda -fsycl-targets=%sycl_triple %s -o %t.out
```

Comment:
```suggestion
// RUN: %clangxx -fsycl %s -o %t.out
```
If options are not needed we can skip them